README.md

```diff
@@ -91,13 +184,108 @@ You then need to do a second call to the `signIn` mutation providing only the `o
 }
 ```
 
-## OAuth
+## OAuth / Login with Sorare
+
+If you want to make requests on behalf of other users you can rely on the [OAuth](https://oauth.net/2/) protocol.
+
+Please reach out to us first through our [Help Center](https://help.sorare.com/hc/en-us/requests/new) and ask us to generate you some OAuth credentials.
+
+We need:
+
+- the name of your App
+- the callback URIs of your OAuth applications
+  - ex: `http://localhost:3000/auth/sorare/callback` (for dev) & `https://myapp.com/auth/sorare/callback` (for prod)
+- the PNG version of your App logo
+
+### OAuth Credentials
+
+Once manually validated, we'll provide you with:
+
+- OAuth UID (or Client ID)
+- OAuth Secret (keep this secret!)
+
+### OAuth Scopes
+
+No specific scopes are available at the moment. All OAuth apps will be have the same privileges.
+
+They are allowed to:
+
+- Access user's nickname, avatar and wallet address
+- Access details about user's Cards, achievements and favourites
+- Access details about user's auctions, offers and notifications
+
+They are NOT allowed to:
```

Comment:
The following are not accessible:
- Email addresses
- Future lineups and rewards
- Claiming rewards
- Bidding, selling, or making offers cards
- Accepting offers or initiating withdrawals